src/index.html

```diff
@@ -9,10 +9,80 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="./styles/style.css"
     />
```

Comment:
Keep your attributes correctly formatted. This includes the 'link' element where each attribute should be on its own line, indented correctly.